modules/1_bastion/versions.tf

```diff
@@ -32,6 +32,9 @@ terraform {
       source  = "hashicorp/random"
       version = "~> 3.4"
     }
+    time = {
+      source = "hashicorp/time"
+      version = "0.9.1"
```

Comment:
This is a new dependency to facilitate synchronization between null_resource providers